src/query/service/src/pipelines/executor/executor_graph.rs

```diff
@@ -274,6 +278,22 @@ impl ExecutingGraph {
             }
 
             if let Some(schedule_index) = need_schedule_nodes.pop_front() {
+                let current_node = locker.graph[schedule_index].clone();
+                let rt_filters = current_node.processor.get_runtime_filters()?;
```

Comment:
Why add runtime filter logic to the exectuor? Can we add it in the pipeline build phase?